File: Count of positives - sum of negatives.dart
List<int> countPositivesSumNegatives(List<int>? input) {
  if (input == null || input.isEmpty) {
    return [];
  } else {
    List<int> total = [];
    
    int positivos = input.where((num) => num > 0).length;
    total.add(positivos);
    
    int sumNegativo = input.where((num) => num < 0).isNotEmpty ? input.where((num) => num < 0).reduce((a, b) => a + b) : 0;
    total.add(sumNegativo);
    
    return total;
  }
}

File: Count of positives -sum of negatives/Count of positives -sum of negatives.ts
export function countPositivesSumNegatives(input: any) {
   if(!input || input.length==0){return []}
   let plus = 0;
   let minus = 0;
   for(let i of input){
     if(i>0){plus++}
     else if(i <0){
       minus+=i
     }
   }
  return [plus,minus]
}
